dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -105,8 +105,8 @@
  */
 @JsonTypeName("file")
 public class FileAppenderFactory<E extends DeferredProcessingAware> extends AbstractAppenderFactory<E> {
-    @NotNull
-    private String currentLogFilename;
+
+	private String currentLogFilename;
```

Comment:
Hi. yeah .. no idea why that happened. My eclipse is configured to use spaces but I guess Mac just ignores that and does whatever :/ I went through it manually and replaced it with spaces where I saw tabs.